README.md

```diff
@@ -97,37 +95,44 @@ PresentationModel instance is automatically retained during configuration change
 Lifecycle callbacks:
 - `onCreate()` — Called when the PresentationModel is created. Initialize your Rx chains in this method.
 - `onBind()` — Called when the View binds to the PresentationModel.
+- `onResume` - Called when the View resumes and begins to receive updates from states and commands.
```

Comment:
Called when the View resumes and starts to receive updates from states and commands.